lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -1785,6 +1785,7 @@ HRESULT DxilShaderReflection::GetDesc(D3D12_SHADER_DESC *pDesc) {
   const ShaderModel *pSM = M.GetShaderModel();
 
   pDesc->Version = EncodeVersion(pSM->GetKind(), pSM->GetMajor(), pSM->GetMinor());
+
   // Unset:  LPCSTR                  Creator;                     // Creator string
```

Comment:
Rationale for not filling in more stuff:
- Our current `!llvm.ident` is `clang version 3.7 (tags/RELEASE_370/final)`, and that's probably not useful.
- Converting compilation flags to FXC's format doesn't look all that useful, let's wait to see if anyone depends on this.
- Instruction counts made sense in a world where shader models defined hard limits on them, which I'm not sure is still the case, and in any case they would probably mean something different between FXC and DXC.